test/e2e_node/podresources_test.go

```diff
@@ -861,7 +863,59 @@ var _ = SIGDescribe("POD Resources [Serial] [Feature:PodResources][NodeFeature:P
 			ginkgo.By("Ensuring the metrics match the expectations a few more times")
 			gomega.Consistently(ctx, getPodResourcesMetrics, 1*time.Minute, 15*time.Second).Should(matchResourceMetrics)
 		})
+	})
+
+	ginkgo.Context("with the rate limit values configured", func() {
+		tempSetCurrentKubeletConfig(f, func(ctx context.Context, initialConfig *kubeletconfig.KubeletConfiguration) {
+			// we intentionally set very low limits to be reasonnably sure to hit them during the tests
```

Comment:
`reasonnably` spelling

Reply:
thanks, will be fixed in the next upload